### DISCUSSION TITLE
How to log when using custom logger

### DISCUSSION THREAD

**Discussion by e-tip** [jmcdo29/nest-commander#581] [Q&A] [answered]
Hi, sorry for my noob question.
I'm using the CommandRunner to run some commands in the cli.
I've used a custom logger we will call CustomLogger and i've passed it to the command like this:
````
async function bootstrap() {
  await CommandFactory.run(
    AppModule,
    CustomLoggerModule.createLogger({
      console: true,
      level: Level.info,
    }),
  );
}
```
createLogger is a static method on the module that returns a CustomLogger ( which is the class that extends the LoggerService ).

My issue is, as the CustomLogger is passed to the run command, if i need to use the logger inside the command, how can i access it ?

**Comment by jmcdo29** [comment, marked as answer]
Just like you would need in a server application, you would need to tell Nest about the `Logger` dependency in the related module and inject via the constructor

**Reply by e-tip** [reply]
I said it was a Noob question. Works perfectly
Thanks
